functions/.directory

```diff
@@ -0,0 +1,3 @@
+[Dolphin]
```

Comment:
Is this file necessary for the development? Is it an IDE file?

Reply:
no it's related to the OS, I'll add it to .gitignore